### PR TITLE
Kit Quenching

### PR DESCRIPTION
**Pull request by AsaphHall**
Addition of the Quenching with Nozzle

### CODE REVIEW

**Comment by RoteKekse** [comment]
grafik

File: peroman.owl [hunk 1]
```diff
@@ -14165,6 +14240,296 @@ Date: 31.03.2023</obo:IAO_0000119>
...
+    <owl:Class rdf:about="https://purl.archive.org/tfsco/TFSCO/TFSCO_00003304">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/PATO_0000052"/>
+        <rdfs:label xml:lang="de">nozzle shape</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- https://purl.archive.org/tfsco/TFSCO/TFSCO_00003305 -->
+
+    <owl:Class rdf:about="https://purl.archive.org/tfsco/TFSCO/TFSCO_00003305">
```
Comment (RoteKekse) [inline peroman.owl:14395]: did you add this class?

Comment (RoteKekse) [reply peroman.owl:14395]: it is not in the image right?

File: peroman.owl [hunk 2]
```diff
@@ -14690,5 +15055,5 @@ Date: 31.03.2023</obo:IAO_0000119>
 
 
 
-<!-- Generated by the OWL API (version 4.5.29.2024-05-13T12:11:03Z) https://github.com/owlcs/owlapi -->
+<!-- Generated by the OWL API (version 4.5.9.2019-02-01T07:24:44Z) https://github.com/owlcs/owlapi -->
```
Comment (RoteKekse) [inline peroman.owl:15058]: what happend here :)?

File: peroman.owl (hunk 1 above)
Comment (AsaphHall) [reply peroman.owl:14395]: No, it is not in the image. This is 'angle' the upper class of 'nozzle spray angle'. It is in the wrong spot in the ontology file though. Sometimes protegé mixes up places. I moved it under 'information content entity'

**Pull request merged**